### PR TITLE
Fix vertical align in nav utils

### LINKED ISSUE #24: Bad vertical align of nav items
Search field is much lower than Explore and Categories. Explore and Categories aren't vertically centered, there is more space on the bottom than on the top.
snimek obrazovky 2018-01-30 v 17 33 55

### PR DESCRIPTION
**Pull request by vasimi**
fixed #24

### CODE REVIEW

**Comment by MikkCZ** [comment]
The diff seems pretty huge. Try `sass --style compact --sourcemap=auto ./sass/style.scss style.css`.

File: themes/frontierline/style.css [hunk 1]
```diff
@@ -18,729 +18,3023 @@ Domain Path: /languages/
 *
 * Don't edit style.css directly!
 **********/
-@font-face { font-family: 'Zilla Slab'; font-weight: normal; font-style: normal; src: url("fonts/ZillaSlab-Regular.woff2") format("woff2"), url("fonts/ZillaSlab-Regular.woff") format("woff"); }
```
Comment (MikkCZ) [inline themes/frontierline/style.css:21]: The diff is huge. Try `sass --style compact --sourcemap=auto ./sass/style.scss style.css` in the theme root. @craigcook can you add the correct sass command in the readme, please? I am not sure if this one is the best to use either.

**Review by MikkCZ — CHANGES_REQUESTED** [review]
Please regenerate the `style.css` file in a way the diff in not that large. LGTM otherwise.

File: themes/frontierline/style.css (hunk 1 above)
Comment (craigcook) [reply themes/frontierline/style.css:21]: Correct, it's compact style.

http://sass-lang.com/documentation/file.SASS_REFERENCE.html#output_style

**Comment by craigcook** [comment]
Recompiled Sass to resolve conflicts and merged manually in 66d6875cdbca31d21ce0c194ec9b6b57e724ca08. Thanks!

**Pull request closed without merging**